recipe/meta.yaml

```diff
@@ -127,7 +131,7 @@ test:
     # - test ! -f $PREFIX/lib/libgandiva.a       # [unix]
     - if exist %PREFIX%\\Library\\lib\\arrow_static.lib exit 1         # [win]
     - if exist %PREFIX%\\Library\\lib\\arrow_flight_static.lib exit 1  # [win]
-    - if exist %PREFIX%\\Library\\lib\\arrow_python_static.lib exit 1  # [win]
+    # - if exist %PREFIX%\\Library\\lib\\arrow_python_static.lib exit 1  # [win]
     - if exist %PREFIX%\\Library\\lib\\parquet_static.lib exit 1       # [win]
     # - if exist %PREFIX%\\Library\\lib\\gandiva_static.lib exit 1       # [win]
 
```

Comment:
Missing `about/description`

Reply:
A `description` was added to the recipe:
https://github.com/AnacondaRecipes/arrow-cpp-feedstock/blob/3174d297567e835d0a260d556286039d6dfb384a/recipe/meta.yaml#L146&L148